recipes-devtools/triton/triton-server_2.58.0.bb

```diff
@@ -1,35 +1,43 @@
 DESCRIPTION = "The Triton Inference Server provides an optimized cloud and edge inferencing solution."
 LICENSE = "BSD-3-Clause"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=1c4cd58940438cc34ebeb4fec0a79ec8"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=cbffd7f72014cdf056bc4e6f176b0c6b"
 
 SRC_URI = "\
-    git://github.com/triton-inference-server/server.git;protocol=https;branch=r22.05 \
+    git://github.com/triton-inference-server/server.git;protocol=https;branch=r25.05;lfs=0 \
     file://0001-Build-fixups.patch \
 "
 
-SRCREV = "be579093ac71ce7b55381ff5747a7b9455739cdf"
+SRCREV = "af417a8ae7457c17ebb44ecda204367f00d11ced"
 
-DEPENDS = "\
+DEPENDS += "\
     triton-core \
     triton-common \
     triton-backend \
     re2 \
+    python3-pybind11 \
 "
 
 COMPATIBLE_MACHINE = "(cuda)"
 
-inherit pkgconfig cmake cuda
+inherit pkgconfig cmake setuptools3 cuda
+
+S = "${WORKDIR}/git"
+B = "${S}"
 
 CXXFLAGS += "${CUDA_CXXFLAGS}"
 
 EXTRA_OECMAKE:append = ' \
     -DCMAKE_INSTALL_PREFIX="${D}${prefix}" \
+    -DCMAKE_PREFIX_PATH=${STAGING_DIR_TARGET}/usr \
+    -DTRITON_VERSION="${PV}" \
 '
 
+SETUPTOOLS_SETUP_PATH = "${B}/src/python"
+
 PACKAGECONFIG ??= "logging http gpu tensorrt"
 PACKAGECONFIG[logging] = "-DTRITON_ENABLE_LOGGING=ON,-DTRITON_ENABLE_LOGGING=OFF"
 PACKAGECONFIG[stats] = "-DTRITON_ENABLE_STATS=ON,-DTRITON_ENABLE_STATS=OFF"
-PACKAGECONFIG[tracing] = "-DTRITON_ENABLE_TRACING=ON,-DTRITON_ENABLE_TRACING=OFF"
+PACKAGECONFIG[tracing] = "-DTRITON_ENABLE_TRACING=ON,-DTRITON_ENABLE_TRACING=OFF, "
```

Comment:
@ichergui was the comma addition here intentional?

Reply:
@dwalkes nope. It was a typo